general/clients/index.md

```diff
@@ -107,7 +107,8 @@ The official Jellyfin Roku app.
 
 **Links:**
 
-- [Roku Channel Store](https://my.roku.com/add/jellyfin).
+<a href='https://my.roku.com/add/jellyfin'><img alt='Get it on the Roku Store' src='https://www.scientology.tv/images/roku-badge.png' width="200"/></a>
```

Comment:
Shouldn't this preferably use an URL from the official Roku site? Unrelated to your change, I think instead of the my.roku.com Link that requires a sign-in, a direct link to the channel store listing might be preferable: https://channelstore.roku.com/details/592369/jellyfin

Reply:
1. I couldn't find one like this on the Roku site. There are logos that could be used from the Roku site but it would look out of place.
2. Ah ok, that link was in the docs before so I thought to just use that link.

Reply:
We can include the image in the repository instead.

Reply:
Ok updated the logo. Honestly doesn't look as bad as I was anticipating but still looks kinda out of place. 

Can understand why you want it from the original repo though.